langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/common/AzureOpenAiChatModelIT.java

```diff
@@ -0,0 +1,25 @@
+package dev.langchain4j.model.azure.common;
+
+import dev.langchain4j.model.azure.AzureOpenAiChatModel;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.chat.common.ChatLanguageModelIT;
+
+import java.util.List;
+
+class AzureOpenAiChatModelIT extends ChatLanguageModelIT {
+
+    @Override
+    protected List<ChatLanguageModel> models() {
+        return List.of(
+                AzureOpenAiChatModel.builder()
+                        .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                        .apiKey(System.getenv("AZURE_OPENAI_KEY"))
+                        .deploymentName("gpt-4o-mini")
+                        .build()
```

Comment:
Seems like model initialization code is extensively duplicated in tests. Would it be possible to create a singleton object responsible for instantiating and caching chat models used in tests?